server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/SearchAction.java

```diff
@@ -78,6 +90,7 @@ public void define(WebService.NewController controller) {
       .setSince("5.2")
       .setDescription("Search quality profiles")
       .setHandler(this)
+      .setChangelog(new Change("6.5", format("The parameters '%s', '%s' and '%s' can be combined without any constraint", PARAM_DEFAULTS, PARAM_PROJECT, PARAM_LANGUAGE)))
```

Comment:
The Change (or at least the String) could be created once as a private static field since it's immutable.